### DISCUSSION TITLE
A way to reset angle?

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#221] [Q&A] [answered]
When displaying the angle on the controller from 0 - 360 degrees is there a way to "zero it".

I usually use a coaxial indicator in the chuck to centre my tools and its quite handy to use the angle to reference 0, 90, 180, 270 degrees.

But sometimes the they arnt in ideal locations

**Comment by kachurovskiy** [comment, marked as answer]
Just press the Stop button to zero the angle.

**Reply by Festivejelly** [reply]
Doh I dont know why I didnt even think of that! (I think I need a coffee refill) Thanks! This makes it muuuuch easier.
